### PR TITLE
Improve product-tests README

### PR DESCRIPTION
**Pull request by amrutagokhale**
Added some text in product-tests `README`:
1) Re-formatted the text that describes different profile specific test groups.
2) Documented LDAP profile.

I am planning to push first commit upstream. Second commit won't go in though since it contains LDAP related documentation.

@anusudarsan @sanjay990

### CODE REVIEW

**Comment by anusudarsan** [comment]
LGTM

File: presto-product-tests/README.md [hunk 1]
```diff
@@ -147,6 +147,8 @@ where [profile](#profile) is one of either:
  installation running on a single Docker container and a single node
  installation of kerberized Presto also running on a single Docker container.
  This profile runs Kerberos without impersonation.
+- **singlenode-ldap** - OpenLDAP server installed on top of the environment
+ in [singlenode](#singlenode) profile.
```
Comment (sanjay990) [inline presto-product-tests/README.md]: This is not completely correct. We modify the singlenode profile. I mean for single-node we have `cdh5-hive` and for ldap we have `centos6-java8-oracle-ldap`. I mean we also include certificates which is not a part of single-node.

Comment (amrutagokhale) [reply presto-product-tests/README.md]: After our discussion on Slack, this is what I changed it to. Let me know how it looks: `Three single node docker containers, one running OpenLDAP server, another with SSL/TLS certificates installed on top of a single node Presto
installation, and third one with a pseudo-distributed Hadoop installation`

**Comment by amrutagokhale** [comment]
I added some text about `authorization` test group in the first commit.

File: presto-product-tests/README.md [hunk 2]
```diff
@@ -147,6 +147,9 @@ where [profile](#profile) is one of either:
  installation running on a single Docker container and a single node
  installation of kerberized Presto also running on a single Docker container.
  This profile runs Kerberos without impersonation.
+- **singlenode-ldap** - Three single node Docker containers, one running OpenLDAP
+ server, another with SSL/TLS certificates installed on top of a single node Presto
+ installation, and third one with a pseudo-distributed Hadoop installation
```
Comment (sanjay990) [inline presto-product-tests/README.md:152]: Three single node Docker containers, one running an OpenLDAP
server, one running with SSL/TLS certificates installed on top of a single node Presto
installation, and one with a pseudo-distributed Hadoop installation.

**Comment by sanjay990** [comment]
LGTM

**Comment by amrutagokhale** [comment]
Cherry-picked first commit to sprint-38. Second commit will go to ldap branch.

**Pull request closed without merging**